### ISSUE TITLE
 [UPGRADE] Upgrade component to the latest version: Elasticsearch OSS

### ISSUE THREAD

**Issue by rafzei** [epiphany-platform/epiphany#1859]
**Is your feature request related to a problem? Please describe.**
- [CVE-2020-7020](https://nvd.nist.gov/vuln/search/results?form_type=Advanced&results_type=overview&search_type=all&cpe_vendor=cpe%3A%2F%3Aelastic&cpe_product=cpe%3A%2F%3A%3Aelasticsearch&cpe_version=cpe%3A%2F%3Aelastic%3Aelasticsearch%3A7.6.2)

**Describe the solution you'd like**
- Update to 7.9.3 or newer if available

**Comment by atsikham** [comment]
Take a look into similar task #1140 that we had previously and its changes. Check ODFE compatibility and upgrade if necessary - https://www.elastic.co/support/matrix#matrix_compatibility.

**Comment by toszo** [comment]
I just received a request to upgrade to version 7.10+

**Comment by przemyslavic** [comment]
@rpudlowski93 Ansible fails after removing the elasticsearch role.
```
2021-03-19T14:25:58.0024252Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - Running: "/ s h a r e d / b u i l d / d e v a w s c e n t c a l i c o / a n s i b l e / e l a s t i c s e a r c h . y m l"machine-cent[0m
2021-03-19T14:25:58.5325362Zmachine-cent[33;21m14:25:58 WARNING cli.engine.ansible.AnsibleCommand - [DEPRECATION WARNING]: ALLOW_WORLD_READABLE_TMPFILES option, moved to a permachine-cent[0m
2021-03-19T14:25:58.5327271Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - plugin approach that is more flexible, use mostly the same config will work,machine-cent[0m
2021-03-19T14:25:58.5332252Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - but now controlled from the plugin itself and not using the general constant.machine-cent[0m
2021-03-19T14:25:58.5339008Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - instead. This feature will be removed from ansible-base in version 2.14.machine-cent[0m
2021-03-19T14:25:58.5346519Zmachine-cent[33;21m14:25:58 WARNING cli.engine.ansible.AnsibleCommand - Deprecation warnings can be disabled by setting deprecation_warnings=False inmachine-cent[0m
2021-03-19T14:25:58.5351758Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - ansible.cfg.machine-cent[0m
2021-03-19T14:25:58.6312665Zmachine-cent[31;21m14:25:58 ERROR cli.engine.ansible.AnsibleCommand - ERROR! the role 'elasticsearch' was not found in /shared/build/devawscentcalico/ansible/roles:/home/epiuser/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/shared/build/devawscentcalico/ansiblemachine-cent[0m
2021-03-19T14:25:58.6318368Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand -machine-cent[0m
2021-03-19T14:25:58.6319313Zmachine-cent[31;21m14:25:58 ERROR cli.engine.ansible.AnsibleCommand - The error appears to be in '/shared/build/devawscentcalico/ansible/elasticsearch.yml': line 8, column 7, but maymachine-cent[0m
2021-03-19T14:25:58.6320271Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - be elsewhere in the file depending on the exact syntax problem.machine-cent[0m
2021-03-19T14:25:58.6320970Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand -machine-cent[0m
2021-03-19T14:25:58.6322182Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand - The offending line appears to be:machine-cent[0m
2021-03-19T14:25:58.6322847Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand -machine-cent[0m
2021-03-19T14:25:58.6323481Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand -   roles:machine-cent[0m
2021-03-19T14:25:58.6324153Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand -     - elasticsearchmachine-cent[0m
2021-03-19T14:25:58.6324781Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleCommand -       ^ heremachine-cent[0m
2021-03-19T14:25:58.6876149Zmachine-cent[38;21m14:25:58 INFO cli.engine.ansible.AnsibleRunner - Run done in 1562543msmachine-cent[0m
```
It is possible that this is because the role is still enabled in the feature mapping.

**Comment by rpudlowski93** [comment]
Fixed in the PR: #2146

**Comment by przemyslavic** [comment]
Fixed.
Tested:
✔ `epicli apply` with 1 `elasticsearch` node
✔ `epicli apply` with 2 clustered `elasticsearch` nodes
✔ `epicli upgrade` with 2 clustered `elasticsearch` nodes: upgraded `Elasticsearch OSS v7.9.1` / `OpenDistro v1.10.1` (epicli v0.9) to `Elasticsearch OSS v7.10.2` / `OpenDistro v1.13.x` (dev)
